socorro/processor/rules/mozilla.py

```diff
@@ -38,16 +38,31 @@ class CopyFromRawCrashRule(Rule):
 
     """
 
-    FLAGS = [
-        ("WindowsErrorReporting", "windows_error_reporting"),
+    FIELDS = [
+        # type, crash annotation name, processed crash key
+        # windows error reporting flag
+        ("flag", "WindowsErrorReporting", "windows_error_reporting"),
+        # Mac memory pressure annotations
+        ("int", "MacMemoryPressureSysctl", "mac_memory_pressure_sysctl"),
+        ("int", "MacAvailableMemorySysctl", "mac_available_memory_sysctl"),
+        ("string", "MacMemoryPressure", "mac_memory_pressure"),
+        ("string", "MacMemoryPressureNormalTime", "mac_memory_pressure_normal_time"),
+        ("string", "MacMemoryPressureWarningTime", "mac_memory_pressure_warning_time"),
+        (
+            "string",
+            "MacMemoryPressureCriticalTime",
+            "mac_memory_pressure_critical_time",
```

Comment:
I looked at writing code that converted CamelCase to snake_case, but crash annotation names are all over the place and I was pretty sure I was going to hit edge cases. I looked for libraries and didn't see any I liked. Then I decided it's better to be able to grep through the code for specific field names and find all the places the field exists than it is to automate generating the name.